packages/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -1518,7 +1555,36 @@ export default class Meeting extends StatelessWebexPlugin {
    * @memberof Meeting
    */
   setupLocusControlsListener() {
-    this.locusInfo.on(LOCUSINFO.EVENTS.CONTROLS_RECORDING_UPDATED,
+    this.locusInfo.on(
+      LOCUSINFO.EVENTS.CONTROLS_MEETING_TRANSCRIBE_UPDATED,
+      ({caption, transcribing}) => {
+        if (transcribing && this.transcription && this.config.receiveTranscription) {
+          this.receiveTranscription();
+        }
+        else if (!transcribing && this.transcription) {
+          Trigger.trigger(
+            this,
+            {
+              file: 'meeting/index',
+              function: 'setupLocusControlsListener',
+            },
+            EVENT_TRIGGERS.MEETING_STOPPED_RECEIVING_TRANSCRIPTION,
+            {caption, transcribing}
+          );
+        }
+        Trigger.trigger(
+          this,
+          {
+            file: 'meeting/index',
+            function: 'setupLocusControlsListener',
+          },
+          transcribing ? EVENT_TRIGGERS.TRANSCRIBING_ON : EVENT_TRIGGERS.TRANSCRIBING_OFF
+        );
```

Comment:
Added this trigger to notify that transcribing has changed 
this works irrespective of the transcription object used by meetings.